Music_Player/Model/PlayController.swift

```diff
@@ -68,7 +68,7 @@ class PlayController: ObservableObject {
             // Source fileを取得する
             let audioFile = try AVAudioFile(forReading: assetURL)
             // PlayerNodeからAudioEngineのoutput先であるmainMixerNodeへ接続する
-            audioEngine.connect(playerNode, to: audioEngine.mainMixerNode, format: nil)
+            audioEngine.connect(playerNode, to: audioEngine.mainMixerNode, format: nil)//ここで落ちる
             // 再生準備
             playerNode.scheduleFile(audioFile, at: nil) {
                 self.moveNextMusic()
```

Comment:
原因は`scheduleFile()`のcomplitionHandlerがメインスレッド以外で実行されるので、`@Published`の`music`などを変更しようとするとエラーになる。
`DispatchQueue.main.async`を使ってメインスレッドで実行されるようにすればOK!
```suggestion
                DispatchQueue.main.async {
                    self.moveNextMusic()
                }
```